src/ImageSharp/Colors/PackedPixel/IPixel.cs

```diff
@@ -0,0 +1,68 @@
+﻿namespace ImageSharp
+{
+    using System;
+    using System.Numerics;
+
+    public interface IPixel
```

Comment:
Browsing it online it seemed to me they use the generic interface only in reflection code. Have not checked with VS yet.